src/views/Settings.vue

```diff
@@ -209,12 +203,13 @@ import TimeZoneModal from '@/views/timezone-modal.vue'
 import { UserService } from '@/services/UserService';
 import { showToast } from '@/utils';
 import { hasError } from '@/adapter';
-import { translate } from '@hotwax/dxp-components';
+import { DxpOmsInstanceNavigator, translate } from '@hotwax/dxp-components';
 import logger from '@/logger';
 import { Actions, hasPermission } from '@/authorization'
 import { DateTime } from 'luxon';
```

Comment:
Remove import for DateTime as not used

Reply:
DateTime is used in "getFacilityOrderCount()" method.